packages/newsroom-signup/src/NewsroomProfile/LearnMoreButton.tsx

```diff
@@ -78,45 +78,78 @@ export class LearnMoreButton extends React.Component<LearnMoreButtonProps, Learn
-              several key benefits. It allows you to be part of a community that has vetted your newsroom and deemed it
-              as having a credible journalistic mission. It also allows your content to be featured alongside content
-              from other approved Civil network newsrooms.
+              In this section we'll take you through the steps to join the Civil network. But first, let’s start with a
+              few key terms. In this section we'll take you through the steps to join the Civil network. But first,
+              let’s start with a few key terms.
             </Paragraph>
+
+            <SectionHeader>What is the Civil Registry</SectionHeader>
             <Paragraph>
-              When creating your Newsroom Profile, we will ask you to complete the application asking for information
-              about your newsroom. We ask that any newsroom applying to the registry provide the information to the best
-              of their ability.
+              The Civil Registry is the online destination where prospective Civil Newsrooms apply to be part of the
+              network.
             </Paragraph>
-            <SectionHeader>What is the Civil Registry</SectionHeader>
             <Paragraph>
-              Civil Registry is the destination to which prospective Civil Newsrooms must apply to access publishing
-              rights on the Civil platform. You can apply to the Civil Registry after creating your Newsroom Smart
-              Contract.
+              Newsrooms on the Civil Registry have been vetted by the community and deemed it as having a credible,
+              journalistic practices as outlined in the{" "}
+              <a href="https://civil.co/constitution/" target="_blank">
+                Civil Constitution
+              </a>.
             </Paragraph>
+
+            <SectionHeader>Civil Tokens</SectionHeader>
             <Paragraph>
-              Being included in the Civil Registry means that the community has vetted a Newsroom and deemed it as
-              having a credible, journalistic mission, and that it has pledged to adhere to the journalistic ethics
-              outlined by the Civil Constitution.
+              Civil tokens (CVL) enable the "community-owned and operated" Civil model to work. A token is a piece of
+              software, it’s not a physical coin. But it does have value and represents a piece of ownership in the
+              network.
             </Paragraph>
+            <Paragraph>
+              For those that want to get more technical, CVL is a utility token based on the ERC20 protocol. It’s a
+              value stored in a decentralized database that’s managed by Civil’s smart contracts, which allows the Civil
+              platform to interact with the Civil token. CVL is the software that bridges Civil with the Ethereum
+              blockchain.
```

Comment:
um, i don't think this paragraph is very good or clear or even totally accurate.  I guess this isn't the place for that.

Reply:
rather than "allow" its probably more acurate to say "control how the civil platform interacts with the token."   and I don't think the bridge line makes any sense as it doesnt really serve that purpose and thats not its reason for being.

Reply:
okay let's remove it and just make it the first paragraph